packages/livekit-rtc/src/participant.ts

```diff
@@ -301,13 +301,13 @@ export class LocalParticipant extends Participant {
     const writableStream = new WritableStream<string>({
       // Implement the sink
       async write(text) {
-        for (const textChunk in splitUtf8(text, STREAM_CHUNK_SIZE)) {
+        for (const textByteChunk of splitUtf8(text, STREAM_CHUNK_SIZE)) {
```

Comment:
the `in` vs `of` bug here is the same as https://github.com/livekit/client-sdk-js/pull/1412